app/helpers/ops_helper.rb

```diff
@@ -27,13 +27,13 @@ def selected?(tree_node, key)
 
   def advanced_tab_warning
     if selected?(x_node, "svr")
-      _('Caution: Manual changes to configuration files can disable the Server!') << " " <<
+      _('Caution: Manual changes to configuration files can disable the Server!') + " " +
         _('Changes made to any individual settings will overwrite settings inherited from the Zone!')
```

Comment:
Someone is going to come along and try to change this back - `<<` is generally preferable over `+`, but interpolation is even _more_ preferable than `<<`.  I think we should change this to the following even though it's arguably uglier

```suggestion
      "#{_('Caution: Manual changes to configuration files can disable the Server!')} #{_('Changes made to any individual settings will overwrite settings inherited from the Zone!')}"
```

Reply:
Me and @jrafanie tried that first but for some reason the strings wouldn't translate when interpolated in that way

Reply:
Ah, yes, sorry, @MelsHyrule and I discussed this and think we should probably just merge the two strings as a single msgid so I suggested let's just do the smallest change possible to fix the issue with modifying frozen strings and worry about changing how we do these messages as a followup.